version.go

```diff
@@ -29,7 +30,13 @@ func ChooseVersions(cfg *config.Config, osd *osd.OSD) (err error) {
 func setupVersion(cfg *config.Config, osd *osd.OSD) (err error) {
 	if cfg.MajorTarget == 0 && cfg.MinorTarget == 0 {
 		// use defaults if no version targets
-		if cfg.ClusterVersion, err = osd.DefaultVersion(); err == nil {
+		if cfg.ClusterVersion, _, err = upgrade.LatestRelease(cfg, cfg.TargetStream); err == nil {
+			if !strings.Contains(cfg.ClusterVersion, "openshift-v") {
+				log.Println("Version didn't have prefix. Adding....")
```

Comment:
Would be good to know which version didn't have the prefix.

Reply:
added in new func